dev/import-beats-resources/kafka/docs/README.md

```diff
@@ -0,0 +1,48 @@
+# Kafka integration
+
+This integration collects logs and metrics from [https://kafka.apache.org](Kafka) servers.
+
+## Compatibility
+
+The `log` dataset module is tested with logs from Kafka 0.9, 1.1.0 and 2.0.0.
+
+The `broker`, `consumergroup` and `partition` metricsets are tested with Kafka 0.10.2.1, 1.1.0, 2.1.1, and 2.2.2.
+
+<!-- TODO: Add a link to Jolokia "input" in Metricbeat -->
```

Comment:
This shows up in the UI, but can also be fixed later.

Reply:
I have fixed this and a couple of typos.